astroquery/astrometry_net/tests/test_astrometry_net.py

```diff
@@ -0,0 +1,205 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import os
+import json
+
+import pytest
+import six
+
+from ...utils.testing_tools import MockResponse
+from ...exceptions import (InvalidQueryError)
+
+from .. import AstrometryNet
+
+_pytest_major, _pytest_minor, _ = pytest.__version__.split('.')
+_pytest_major = int(_pytest_major)
+_pytest_minor = int(_pytest_minor)
+_pytest_is_old = not ((_pytest_major == 3) and (_pytest_minor >= 3))
```

Comment:
Please use LooseVersion instead of these 4 lines. This way thinks that my pytest 4.5 is too old.